apps/system/js/app_window.js

```diff
@@ -1881,6 +1883,19 @@
     }
   };
 
+  AppWindow.prototype._handle__created = function() {
```

Comment:
nit: I think we could put this in this.installSubModules(); then we don't need to listen to _created.